pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -357,6 +359,10 @@ func (a *Bootstrap) addParentFiles(dependencies asset.Parents) {
 		dependencies.Get(asset)
 		a.Config.Storage.Files = append(a.Config.Storage.Files, ignition.FilesFromAsset(rootDir, 0600, asset)...)
 	}
+
+	journalCertKey := &tls.JournalCA{}
```

Comment:
this feels out of place in `addSystemdUnits` it seems like it belongs here https://github.com/openshift/installer/pull/967/commits/907a22db51e9453462416736476262ba1c53906a#diff-2d9b655c8052ca2e3bdd9004778c75bcR357

Reply:
It can't go there because the key and cert need to be readable by journal-gatewayd (which doesn't run as root). I should probably also change the owner of these files... I hope that works on RHCOS...